src/controllers/translation.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import { MiddleWare } from '../types';
+import { IGBO_TO_ENGLISH_API, MAIN_KEY } from '../config';
+
+interface TranslationMetadata {
+  igbo: string;
```

Comment:
I think we would want to have a schema similar to something like the following:
``` suggestion
  text: string;
  language: LanguageEnum
```

`LanguageEnum` would include he ISO language codes, so for Igbo it would be `ibo`

Reply:
I'm now validating the request body. For now, the call to the ML endpoint won't change but that's more of an implementation detail.